tests/test_datepicker.py

```
@@ -400,10 +401,14 @@ def setUpClass(cls) -> None:
         cls.browser = launch_certification_browser(cls.playwright)
         cls.addClassCleanup(cls.browser.close)
 
-    def open_fixture(self, case=CERTIFICATION_CASES[0]):
+    def open_fixture(self, case=CERTIFICATION_CASES[0], *, fixed_time: str | None = None):
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `open_fixture`

(ANN202)